lib/tutorials/en_US/getting-started.md

```diff
@@ -18,23 +19,25 @@ The most basic server looks like the following:
 
 const Hapi = require('hapi');
 
-const server = new Hapi.Server();
-server.connection({ port: 3000, host: 'localhost' });
+const server = Hapi.server({ port: 3000, host: 'localhost' });
 
-server.start((err) => {
-
-    if (err) {
-        throw err;
-    }
+const init = async () => {
+    await server.start();
     console.log(`Server running at: ${server.info.uri}`);
+};
+
+init().catch((err) => {
+    console.log(err);
+    process.exit(1);
 });
+
 ```
 
-First, we require hapi. Then we create a new hapi server object. After that we add a connection to the server,  passing in a port
-number to listen on. After that, start the server and log that it's running.
+First, we require hapi. Then we create a new hapi server object with a configuration object containing a host and a port
+number to listen on. After that we start the server and log that it's running.
 
-When adding the server connection, we can also provide a hostname, IP address, or even
-a Unix socket file, or Windows named pipe to bind the server to. For more details, see [the API reference](/api/#serverconnectionoptions).
+When creating a server, we can also provide a hostname, IP address, or even
+a Unix socket file, or Windows named pipe to bind the server to. For more details, see [the API reference](/api/#server.options).
```

Comment:
The URL seems to be `#-server-options`

Reply:
I'm not sure this is correct

Reply:
![screen shot 2018-03-07 at 09 08 50](https://user-images.githubusercontent.com/916064/37069720-2efe63ea-21e7-11e8-96df-dee17480b9c9.png)